influxdb/src/main/scala/akka/stream/alpakka/influxdb/InfluxDBSettings.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.influxdb
+
+import java.util.concurrent.TimeUnit
+
+import scala.concurrent.duration.TimeUnit
+
+object InfluxDBSettings {
```

Comment:
We often see the need for specific settings for flows vs sources. Might be better to start off with both `InfluxDbSourceSettings` and `InfluxDbWriteSetting`.

Reply:
Refactored to InfluxDbReadSettings and InfluxDbWriteSettings